pkgs/development/python-modules/simplefix/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, python, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "simplefix";
+  version = "1.0.12";
+
+  src = fetchFromGitHub {
+    repo = "simplefix";
+    owner = "da4089";
+    rev = "v${version}";
+    sha256 = "0pnyqxpki1ija0kha7axi6irgiifcz4w77libagkv46b1z11cc4r";
+  };
+
+  checkPhase = ''
+    cd test
+    ${python.interpreter} -m unittest all
+  '';
+
+  meta = {
```

personal preference, don't feel strongly about this.
```suggestion
  meta = with lib; {
```